p11-kit/conf.c

```diff
@@ -228,9 +228,15 @@ _p11_conf_load_globals (const char *system_conf, const char *user_conf,
 		goto finished;
 	}
 
-	if (mode != CONF_USER_NONE && getauxval (AT_SECURE)) {
-		p11_debug ("skipping user config in setuid or setgid program");
-		mode = CONF_USER_NONE;
+	if (mode != CONF_USER_NONE) {
+		if (getauxval (AT_SECURE)) {
+			p11_debug ("skipping user config in setuid or setgid program");
+			mode = CONF_USER_NONE;
+		} else if (getenv ("P11_NO_USER_CONFIG")) {
```

Comment:
A couple of minor nits: other envvars used in p11-kit have ```P11_KIT_``` prefix and are accessed through ```secure_getenv``` (though it's equivalent to ```getenv``` here because of the above ```getauxval``` check).